lib/scarpe.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
+require_relative "../logger/logger.rb"
+
 if RUBY_VERSION[0..2] < "3.2"
-  $stderr.puts "Scarpe requires Ruby 3.2 or higher!"
+  ScarpeLogger::Logger.logger.error("Scarpe requires Ruby 3.2 or higher!")
```

Comment:
I love having the multiple levels (error vs warning etc). But I think we'll want something less verbose to call it. Maybe we could add "Scarpe.error()" to call ScarpeLogger::Logger.logger.error() and so on?

Reply:
if not to maintain consistency #127  should be edited as shortest that works?

Reply:
While it's good to refer to Scarpe classes in some consistent way, I don't think we want logging a line to require a really long class name. If we had a top-level alias (e.g. a singleton method on the Scarpe class) then we could have both -- a fully-resolved class name, in this case Scarpe, and a short alias so that logging wasn't painful.

Basically, if we have a short alias here, I don't think that makes a difference to whether we choose relative or absolute class names in #127.